src/models/view/nav-tabs.ts

```diff
@@ -66,6 +67,14 @@ export const NavTabSectionModel =
        ];
       return (deletableTypes.includes(self.type) && userOwnsDocument);
     },
+    get allowDelete(){
```

Comment:
Unless it is used somewhere else, you should remove the `showDeleteForUser` above.